cumulus/parachains/runtimes/test-utils/src/lib.rs

```diff
@@ -534,9 +534,9 @@ pub fn assert_metadata<Fungibles, AccountId>(
 	Fungibles: frame_support::traits::fungibles::metadata::Inspect<AccountId>
 		+ frame_support::traits::fungibles::Inspect<AccountId>,
 {
-	assert_eq!(Fungibles::name(asset_id.clone().into()), Vec::from(expected_name),);
-	assert_eq!(Fungibles::symbol(asset_id.clone().into()), Vec::from(expected_symbol),);
-	assert_eq!(Fungibles::decimals(asset_id.into()), expected_decimals);
+	assert_eq!(Fungibles::name(&asset_id.clone().into()), Vec::from(expected_name),);
+	assert_eq!(Fungibles::symbol(&asset_id.clone().into()), Vec::from(expected_symbol),);
```

Comment:
Possible to remove the `.clones` in scenarios like this or?

Reply:
All these tests helper functions provide `asset_id` as `impl Into<Fungibles::AssetId> + Clone`. I would need to change the API of these functions, then a lot of test callsites.
I was thinking it's not worth the effort and the noise since these are only tests.